stacks/serverless/dovetail-cdn-arranger.yml

```diff
@@ -0,0 +1,133 @@
+# stacks/serverless/dovetail-cdn-arranger.yml
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Dovetail CDN Arranger lambda function
+Conditions:
+  IsProduction: !Equals [!Ref EnvironmentType, Production]
+Parameters:
+  CodeS3Bucket:
+    Type: String
+  CodeS3ObjectVersion:
+    Type: String
+  EnvironmentType:
+    Type: String
+  DovetailHost:
+    Type: AWS::SSM::Parameter::Value<String>
+  DovetailToken:
+    Type: AWS::SSM::Parameter::Value<String>
+  DynamodbTableArn:
+    Type: AWS::SSM::Parameter::Value<String>
+Resources:
+  DovetailCdnArrangerWorkspaceBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      # TODO: tag-specific expirations
```

Comment:
I feel like tag-based lifecycles it part of what got us into trouble with the `prx-dovetail` bucket. I would prefer using multiple buckets if we find that this app is managing files with very distinct lifecycle requirements. It also really simplifies cost tracking to have things in different buckets.

Reply:
+1 ... I didn't look into tag-specific expirations because I'm 90% sure I don't care anymore, and expiring everything in this bucket after a certain time is fine.

That said - the lambda still tags everything it PUTs, so we'd have the option of expiring sooner than the "global" expiration.